packages/model-viewer/src/features/scene-graph/model.ts

```diff
@@ -36,11 +38,16 @@ export class Model implements ModelInterface {
     const {gltf, gltfElementMap} = correlatedSceneGraph;
 
     gltf.materials!.forEach(material => {
-      this[$materials].push(new Material(
-          onUpdate,
-          gltf,
-          material,
-          gltfElementMap.get(material) as Set<MeshStandardMaterial>));
+      const correlatedMaterial =
+          gltfElementMap.get(material) as Set<MeshStandardMaterial>;
+
+      if (correlatedMaterial) {
```

Comment:
Our style is to use explicit null checks like `correlatedMaterial != null`.

Reply:
This style is also recommended in c++ to return early on error

Reply:
Done.
I did this to combine the check for null and undefined, but I can add explicit checks

Reply:
ah, so this is why we allow `==` and `!=` for exactly one situation: comparing to `null`, because this one combines the check for `null` and `undefined`. However, it's still better than a truthiness check, because that will also catch things like empty string, zero, etc.

Reply:
Done.